free-2.md

```diff
@@ -2,27 +2,31 @@
 
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+This section describes who the project would serve and why a language might be a good way to meet their needs.
 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL will allow the users to create coloring pages with any pictures
+they have or a picture found on Google randomly from the keywords the users input using this DSL.
```

Comment:
This sounds like a really, really cool program (and if you take graphics maybe suggest this as your final project!) but I don't think it makes for a good DSL. I'm having a hard time seeing how you'll have any language-design choices to make, and instead you'll be bogged down in converting an image to a coloring-book page (probably just edge detection, and then inverting the magnitude of the gradient)